kore/src/Kore/Step/Simplification/SubstitutionSimplifier.hs

```diff
@@ -195,9 +194,7 @@ simplifyAnds MakeAnd { makeAnd } (NonEmpty.sort -> patterns) =
             AndF And { andFirst, andSecond } ->
                 foldM simplifyAnds' intermediate [andFirst, andSecond]
             _ -> do
-                let sideCondition =
-                        SideCondition.topTODO
-                        `SideCondition.andCondition` intermediateCondition
+                let sideCondition = SideCondition.topTODO
```

Comment:
This is fine for now, but let's make a follow-up pull request to add a `SideCondition` argument to `simplifyAnds` and remove this `topTODO`.